PanModal/Controller/PanModalPresentationController.swift

```diff
@@ -263,16 +263,17 @@ open class PanModalPresentationController: UIPresentationController {
         
         if let keyboardSize = notification.userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? CGRect {
             
-            if #available(iOS 11.0, *) {
-                let safeAreaOffset = presentedView.safeAreaInsets.bottom
-                keyboardHeight = keyboardSize.height
-                keyboardHeight -= safeAreaOffset
-            }
+            keyboardHeight = keyboardSize.height
             
             if let scrollView = presentable?.panScrollable, pannedToMax {
-                scrollView.contentInset.bottom = keyboardHeight
-                scrollView.scrollIndicatorInsets.bottom = scrollView.scrollIndicatorInsets.bottom - keyboardHeight
+                
+                var contentInset: UIEdgeInsets = scrollView.contentInset
+                contentInset.bottom = keyboardHeight + 20
```

Comment:
Whats this 20? Can we make it a constant out of scope?

Reply:
@TheiOSDude  just some padding, i can make a constant

Reply:
Yeah please, just saves magic numbers floating.

Reply:
@TheiOSDude added